sources/lib/util/options.ml

```diff
@@ -250,6 +253,28 @@ module M = struct
       Format.eprintf "Args parsing error: unkown SAT solver %S@." s;
       exit 1
 
+  let no_cf = "no"
+  let cf = "yes"
+  let cf_max = "max"
+  let cf_both = "both"
+  let set_cube_strategy str =
+    match String.lowercase_ascii str with
+    | "no" | "n" | "false" | "f" ->
+      cubefast := no_cf
+    | "yes" | "y" | "true" | "t" ->
+      cubefast := cf
+    | "max" | "m" | "maximum" ->
+      cubefast := cf_max
+    | "both" | "b"  ->
+      cubefast := cf_both
+    | _ -> (
+        raise
+          (Arg.Bad
+             ("Bad value '" ^
+              str ^
+              "' for option -cubefast"))
+      )
```

Comment:
Could the `cubefast` option actually be a sum type rather than encode its options in a string ?